lib/tasks/default.rake

```diff
@@ -0,0 +1,3 @@
+task :default do
+  system("bundle exec spinach")
```

Comment:
I don't recall, but Spinach may have a rake task of its own, and you can "invoke" a rake task from a ruby environment without shelling out.